src/UserMenu/index.js

```diff
@@ -53,7 +53,11 @@ const UserMenu = ({ onLogOutClick, onProfileClick, selectedUserProfile, user, us
           <Dropdown.Divider />
         </>}
 
-        <Dropdown.Item onClick={() => cookieSettingsRef.current.click()}>{t('cookieSettingsItem')}</Dropdown.Item>
+        {window.Osano?.cm && (
```

Comment:
Let's add some optional chaining to the window object (like `window?.Osano?.cm`) so we're not set up for future failures in headless test environments.